test/cppsuite/test_harness/workload/database_operation.h

```diff
@@ -236,45 +223,34 @@ class database_operation {
              */
             tc->sleep();
 
-            /* Pick a random collection to update. */
-            {
-                std::shared_ptr<collection> coll = tc->database.get_random_collection();
-                if (!coll) {
-                    debug_print(
-                      "update_operation: no collections found, did you forget to create one ?",
-                      DEBUG_ERROR);
-                    break;
-                }
-                collection_id = coll->id;
+            /* Choose a random collection to update. */
+            collection &coll = tc->database.get_random_collection();
 
-                /* Look for existing cursors in our cursor cache. */
-                if (ccm.find(collection_id) == ccm.end()) {
-                    debug_print("Thread {" + std::to_string(tc->id) +
-                        "} Creating cursor for collection: " + coll->name,
-                      DEBUG_TRACE);
-                    /* Open a cursor for the chosen collection. */
-                    tc->session->open_cursor(
-                      tc->session, coll->name.c_str(), nullptr, nullptr, &cursor);
-                    ccm.emplace(
-                      collection_id, collection_cursor{std::shared_ptr<collection>(coll), cursor});
-                }
+            /* Look for existing cursors in our cursor cache. */
+            if (cursors.find(coll.id) == cursors.end()) {
+                debug_print("Thread {" + std::to_string(tc->id) +
+                    "} Creating cursor for collection: " + coll.name,
```

Comment:
Seems that the formatting is wrong here ?

Reply:
Clang format does that.